modAionImpl/src/org/aion/zero/impl/config/Network.java

```diff
@@ -11,6 +11,7 @@
     MAINNET("mainnet", 256),
     CONQUEST("conquest", 128),
     MASTERY("mastery", 32),
+    AVMTESTNET("avmtestnet", 31), // temporary chainid
```

Comment:
is it still tempoary?

Reply:
yeah, changing the chain id is something that may still happen